pages/your-profile.vue

```diff
@@ -3,32 +3,57 @@ definePageMeta({
   layout: "login-layout",
   middleware: "no-auth",
 });
+
+onMounted(() => {
+  const isStoragePublic = useIsStoragePublic();
+  isStoragePublic.value = false;
+});
```

Comment:
I want the user profile photo was stored in the private bucket so that I make it to switch to private bucket and there is no option to switch to public bucket on your-profile page